### ISSUE TITLE
Vets-api terminal access for Joshua Drumm

### ISSUE THREAD

**Issue by JoshingYou1** [department-of-veterans-affairs/va.gov-team#69030]
### Your Name

Joshua Drumm

### Your GitHub handle (terminal access is managed through GitHub)

@JoshingYou1

### Your dsvagovcloud AWS user name (if applicable)

Joshua.Drumm

### Team, Role, and Company of the target individual

10-10 Team, Software Engineer, Agile Six Applications, Inc.

### Product Manager (PM) name and email

Heather Justice, heather.justice@adhocteam.us

### Product Owner (PO) name and email

Patrick Bateman, patrick.bateman@va.gov

### COR Name (leave "None" if you listed an AWS user name above)

None

### Vendor Onboarding Representative (VOR) Name (leave "None" if you listed an AWS user name above)

None

### Please confirm if you are on the VFS Team Roster or Platform Team Roster

- [X] [VFS Team Roster](https://docs.google.com/spreadsheets/d/11dpCJjhs007uC6CWJI6djy3OAvjB8rHB65m0Yj8HXIw/edit?folder=0ALlyxurHpUilUk9PVA#gid=2042046665)
- [ ] [Platform Team Roster](https://vfs.atlassian.net/wiki/spaces/AP/pages/1908834623/Platform+Roster)

### Please confirm E-QIP Transmittal/Adjudication. This could be a screenshot of your Transmittal/Adjudication email, a link to another request where you've submitted this proof (i.e. AWS/SOCKS), etc.

- [X] I have attached proof that I have either transmitted my E-QIP survey or previously received a favorable decision (by including in "Additional Notes" section below)

### ArgoCD terminal access requested for the following environments

- [X] dev
- [X] staging
- [X] sandbox
- [X] production (if requesting, justify prod access in 'Additional Notes' section)

### Additional Notes

![](https://github.com/department-of-veterans-affairs/va.gov-team/assets/34111449/ac90e855-95ca-4a55-927c-d6f97178e57a)

I need production access for debugging purposes (logs, etc.).

### Additional resources for Support Engineer

- [ ] For users on a VFS team but not in the VFS Team Roster, they must start the [Platform orientation process](https://depo-platform-documentation.scrollhelp.site/getting-started/Platform-Orientation.1877344532.html)
- [ ] Instructions for handling requests [here](https://vfs.atlassian.net/wiki/spaces/PTST/pages/2183037084/Backend+Support+Responsibilities#Handle-vets-api-ArgoCD-terminal-access) - Support Only

**Comment by bdech** [comment]
@JoshingYou1 You’ve been granted access to the `dev`, `staging`, and `sandbox` environments.

Here's some additional documentation for [Vets API on EKS](https://depo-platform-documentation.scrollhelp.site/developer-docs/vets-api-on-eks#VetsAPIonEKS-TerminalAccess) that may help.

Access to `prod` is pending approval by OCTODE, and we'll update you as 
soon as it's available

**Comment by BillChapmanUSDS** [comment]
Sorry,  access justifcation needs to be specific.  You have a specific issue you've had to debug or need to debug.  Access allowed for 3 months with example provided tag removed. if a more specific justification is provided and this it tagged it can be extended to 6 months.

**Comment by bdech** [comment]
@JoshingYou1 You've been granted access `Production`, 
with access to Production limited to 3 months, expiring 2/6/24. If you can provide specifics for access, Bill can approve extended access to 6 months.

Please set a reminder for yourself to return to this Issue and 
request an extension within 1 week of the expiration date. You'll 
receive 1 reminder with a 72 hour window to respond. Failure to 
respond will result in your access being revoked and will require 
you to fill out a new request.

**Comment by bdech** [comment]
Slack reminder [here](https://dsva.slack.com/archives/C02RLAUM2LV/p1699292694500229)

**Comment by lindsay-insco** [comment]
@JoshingYou1 Your Vets API Terminal Production access has expired.

Would you like to request an extension? You have 72 hours to respond, or your
access will be revoked and you will need to fill out a new request

**Comment by JoshingYou1** [comment]
> @JoshingYou1 Your Vets API Terminal Production access has expired.
> 
> Would you like to request an extension? You have 72 hours to respond, or your
> access will be revoked and you will need to fill out a new request

Yes, I would like to request an extension.

**Comment by bdech** [comment]
@LindseySaari @ericboehs can one of you please give us a :thumbsup: to extend their access?

**Comment by bdech** [comment]
@JoshingYou1 Your `production` access has been extended for 6 months,
expiring 09/4/24.

Please set a reminder for yourself to return to this Issue and 
request an extension within 1 week of the expiration date. You'll 
receive 1 reminder with a 48 hour window to respond. Failure to 
respond will result in your access being revoked and will require 
you to fill out a new request.

**Comment by bdech** [comment]
support slack reminder [here](https://dsva.slack.com/archives/C06CY71PCAX/p1709583205696889)

**Comment by JoshingYou1** [comment]
Yes, I'd like an extension please. Let me know if an action is needed on my part. Thank you!

**Comment by bdech** [comment]
I just needed a verbal confirmation. Thanks Joshua.

**Comment by bdech** [comment]
@LindseySaari @ericboehs can one of you please give us a :thumbsup: to extend their access?

**Comment by bdech** [comment]
support slack reminder [here](https://dsva.slack.com/archives/C06CY71PCAX/p1725628414097939)

**Comment by davidpetras** [comment]
@JoshingYou1  Your <Vets API Terminal> Production access has expired.

Would you like to request an extension? You have 72 hours to respond, or your 
access will be revoked and you will need to fill out a new request.

**Comment by davidpetras** [comment]
@LindseySaari @ericboehs can one of you please give us a :thumbsup: to extend their access?

**Comment by bdech** [comment]
@JoshingYou1  your Production access has been extended for 6 months,expiring 9/4/25.

Please set a reminder for yourself to return to this Issue, reopen it, and request an extension within 1 week of the expiration date. You'll receive 1 reminder with a 48 hour window to respond. Failure to respond will result in your access being revoked and will require you to fill out a new request.

**Comment by GWright321** [comment]
@LindseySaari @ericboehs can one of you please give us a 👍 to extend their access?

**Comment by GWright321** [comment]
@JoshingYou1  your Production access has been extended for 6 months,
expiring April 15, 2026.

Please set a reminder for yourself to return to this Issue, reopen it, and
request an extension within 1 week of the expiration date. You'll
receive 1 reminder with a 48 hour window to respond. Failure to
respond will result in your access being revoked and will require
you to fill out a new request.

**Comment by GWright321** [comment]
Slack Reminder [here](https://dsva.slack.com/archives/C06CY71PCAX/p1760541332665339)
